arrays/index.js

```diff
@@ -0,0 +1,20 @@
+// Hobbies
+const hobbies = new Array("water skiing", "bouldering", "running",)
+
+console.log(hobbies.shift())
```

Comment:
This will remove the first thing in the array. So it will change the array. You could use hobbies[0] to access the first element without mutating the array